CRM/Utils/System/Drupal8.php

```diff
@@ -212,6 +212,7 @@ public function resetBreadCrumb() {
 
   /**
    * @inheritDoc
+   * @deprecated
    */
   public function addHTMLHead($header) {
```

Comment:
Not a blocker but suggested addition:

`\CRM_Core_Error::deprecatedFunctionWarning('Civi::resources()->addStyleFile() or addScriptFile() etc');`